README.md

```diff
@@ -1,50 +1,76 @@
+While contributing to the Meshtastic RFC repository, please adhere to our community standards and guidelines. These are in place to ensure a productive and respectful environment for collaboration.
 
-1. **Write up your proposal**: There are two methods to write up your proposal within this repository. If you are comfortable with how to work with Git, we recommend you clone this repository, create a copy of the RFC template in the `/rfcs` directory, and submit a PR. If you are not familiar with Git, we recommend you follow the steps below:
+- Be respectful and constructive in your communication.
+- Clearly articulate your proposals and feedback.
+- Engage in discussions with an open mind.
+- **Informal Discussions**: Before submitting an RFC, you may want to discuss your idea informally. Join us on our [Discord](https://discord.com/invite/ktMAKGBnBs) for preliminary discussions and feedback from the community.
 
-   1. Sign into or create a GitHub account. Then, navigate to [https://github.com/meshtastic/rfcs](https://github.com/meshtastic/rfcs).
+## Applicability
 
-   1. Navigate to the RFC template in the root of the repository. Copy the contents of the template to your clipboard.
+### When an RFC is Required
 
-   1. Navigate to the `/rfcs` directory within this repository on GitHub. Click the `Add file` button and select `Create new file`. Name this file `YYYY-MM-DD-feature-name.md`, replacing `YYYY-MM-DD` with the current date and `feature-name` with a descriptive name for your proposal. Paste the contents of the template into this file.
+An RFC is required for any significant change to Meshtastic, introduces new features, or has the potential to impact the user experience or ecosystem broadly. This includes, but is not limited to:
 
-   1. Fill in the template with your proposal. Be sure to include a descriptive title and summary, as well as a detailed description of your proposal. Once you are done, copy the title and high-level description of your proposal. You will need this in the next step.
+- Changes to the core architecture
+- Introduction of new dependencies
+- Major UI/UX changes
+- Changes affecting backward compatibility
+- Significant performance optimizations
 
-   1. When you are finished, click "Commit Changes" at the top of the page. Enter the title and description from the last step into the appropriate fields. Then, click "Propose changes." This will create a new Pull Request (PR) for your proposal.
+### When an RFC is _Encouraged_
 
-   1. Congratulations! You have successfully submitted an RFC. Now, you can engage in the discussion about your proposal. You can view your PR by navigating to the pull requests tab here: [https://github.com/meshtastic/rfcs/pulls](https://github.com/meshtastic/rfcs/pulls)
+An RFC is encouraged for any initiative where someone or a group of someones wishes to benefit from the collective wisdom and experience of the Meshtastic community, but are **not subject to the rules below**.
```

Comment:
I feel like this is an important section to have, our community is extremely talented and knowledgeable. I would also understand if y'all don't feel it's necessary to call out here.